sw/tests/bare-metal/hostd/mbox_test.c

```diff
@@ -1,6 +1,10 @@
 // Copyright 2023 ETH Zurich and University of Bologna.
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
+//
+// Alessandro Ottaviano <aottaviano@iis.ee.ethz.ch>
+//
+// Simple mailbox test
 
```

Comment:
At some point we should add simple `mbox` drivers instead of doing the writes in the main

Reply:
Hey guys, thanks for the comments. At the very moment, most of the changes I did were just bruteforcing to try and have this task complete as soon as possible, this PR requires a significant cleanup before it can be reviewed.

Reply:
> At some point we should add simple `mbox` drivers instead of doing the writes in the main

I think this diff is a mistake, I am sure a rebase will fix this.